api/util/parseDate.js

```diff
@@ -0,0 +1,10 @@
+const chrono = require('chrono-node');
+const moment = require('moment');
+
+module.exports = (input, offset) => {

```

Comment:
it may help to expand this into something like `translateDateTime` (possibly put this in a helpers folder?) such that you pass it the date string (2 days from now) and the time zone and it returns an object with the offset time (delayAmt), formatted time, as well as the values passed in for reference.

This should clean up the above and help refactor it down to a single responsibility.

thoughts?